docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc

```diff
@@ -116,6 +116,26 @@ Custom yaml settings are silently ignored by {fleet}.
 Settings with input blocks, such as Max agents are still effective.
 ====
 
+[[known-issue-issue-2249-8.6.2]]
+.{agent} does not load {fleet} configuration after upgrading from {agent} version 8.2 or earlier
```

Comment:
```suggestion
.{agent} does not load {fleet} configuration when upgrading to {agent} version 8.6.x from version 8.2 or earlier
```

Based on https://github.com/elastic/ingest-docs/pull/195#discussion_r1185267796 the affected versions still need to be clearer. I want to be extra clear that this only affects upgrades to 8.6.x, even though these are the 8.6 release notes.